org.eclipse.dartboard/plugin.xml

```diff
@@ -134,7 +134,7 @@
         <wizard
              category="org.eclipse.dartboard.eclipse.wizards"
              class="org.eclipse.dartboard.project.DartFileWizard"
-             icon="platform:/plugin/org.eclipse.ui.ide/icons/full/etool16/newfile_wiz.png"
+             icon="assets/img/dart_16.png"
```

Comment:
Same icon used for project and  file wizard

Reply:
I added a new icon based on the generic editor icon with a Dart logo.